lec03-basic-r.Rmd

```diff
@@ -1,4 +1,732 @@
 ---
-title: "Basic R (part 2): Functions, data frames, indexing"
+title: "Manipulating and analyzing data with dplyr"
 ---
 
+
+**Copyright (c) Data Carpentry**
+
+*Note: This lecture content was originally created by voluntary contributions to
+[Data Carpentry](http://datacarpentry.org) and has been modified to
+align with the aims of EEB430. Data Carpentry is an organization focused on data
+literacy, with the objective of teaching skills to researchers to enable them to
+retrieve, view, manipulate, analyze, and store their and other’s data in an open
+and reproducible way in order to extract knowledge from data. For EEB430, we are
+making all our content available under the same license, [The Creative Commons](https://creativecommons.org/), 
+license, so if that anyone in the future can re-use or modify our course content, without infringing on copyright licensing issues.*
+
+
+> ## Learning Objectives
+>
+> * Describe what a data frame is.
+> * Load external data from a .csv file into a data frame in R.
+> * Summarize the contents of a data frame in R.
+> * Understand the purpose of the **`dplyr`** package.
+> * Select certain columns in a data frame with the **`dplyr`** function `select`.
+> * Select certain rows in a data frame according to filtering conditions with the **`dplyr`** function `filter` .
+> * Link the output of one **`dplyr`** function to the input of another function with the 'pipe' operator `%>%`.
+> * Add new columns to a data frame that are functions of existing columns with `mutate`.
+> * Understand the split-apply-combine concept for data analysis.
+> * Use `summarize`, `group_by`, and `tally` to split a data frame into groups of observations, apply a summary statistics for each group, and then combine the results.
+
```

Comment:
Don't forget to include the `Lesson Outline` with approximate times. It's better to overestimate time by a bit then to underestimate. I'll do a more thorough review later today.